gradlew

```diff
@@ -1,164 +1,164 @@
-#!/usr/bin/env bash

```

Comment:
Gradle manages its own files. It's not that I made the modifications, but  the gradle guys made modifications to the generated scripts format. (running gradle wrapper).